matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.css

```diff
@@ -475,6 +475,15 @@ td[name="contact"] {
   white-space: normal;
   line-height: 12px;
 }
+.xTooltip {
+  white-space: pre-wrap;
+  white-space: -moz-pre-wrap;
+  white-space: -pre-wrap;
+  white-space: -o-pre-wrap;
+  word-wrap: break-word;
```

Comment:
You can remove this line too, since it's a workaround for IE<8 not supporting `pre-wrap`. We don't support IE7 or older.